vwifi.c

```diff
@@ -529,7 +532,8 @@ static enum hrtimer_restart vwifi_beacon(struct hrtimer *timer)
         .boottime_ns = ktime_get_boottime_ns(),
         .chan = vif->channel,
     };
-
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 7, 0)
```

Comment:
Rewrite as following:
```c
#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 7, 0)
...
#endif
```

You should explain the above changes in git commit messages.